File: 151002/Redkovskiy/StoryTeller/src/main/java/com/example/storyteller/model/Post.java
package com.example.storyteller.model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "tbl_post")
public class Post {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(length = 2048)
    private String content;

    @ManyToOne()
    @JoinColumn(name = "story_id", referencedColumnName = "id")
    private Story story;
}

File: 151004/Ermolovich/DC_Lab1/DC_Lab1/DC_Lab1/Models/Post.cs
using System;
using System.Collections.Generic;

namespace DC_Lab1.Models;

public partial class Post
{
    public int Id { get; set; }

    public int? storyId { get; set; }

    public string? Content { get; set; }

    public virtual Story? Story { get; set; }
}
